ui/playwright.config.js

```diff
@@ -67,7 +67,7 @@ module.exports = defineConfig({
         ...devices['Desktop Chrome'],
         provider: 'Meshery',
         // Use prepared auth state.
-        storageState: 'playwright/.auth/user.json',
+        storageState: 'playwright/.auth/user-local-provider.json',
```

Comment:
@jerensl How about avoiding repeating the storage state path by defining it once and reusing it ?
Something as:

const storageStateMeshery = 'playwright/.auth/user-meshery-provider.json';
const storageStateNone = 'playwright/.auth/user-local-provider.json';